count_lines.py

```
@@ -0,0 +1,32 @@
+import os
+import re
+
+total_lines = 0
+file_count = 0
+
+def is_executable_line(line):
```

Ruff (0.15.13)

[warning] Too many return statements (7 > 6)

(PLR0911)